### PR TITLE
Hide `last indexed SHA` when a repo could not be indexed yet

### LINKED ISSUE #26336: Hide lastest commit sha when no commit is available
### Description

When I create a new repo without any commit, the `Last indexed SHA` field looks like this:
image

Just an empty button. I think the button should be hidden, if the content is not available. 

### Gitea Version

1.20.2

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

Docker compose

### Database

SQLite

### PR DESCRIPTION
**Pull request by CaiCandong**
Now,  for a new repo without any commit, the Last indexed SHA field looks like this:
Before:
image
After: 
image

fix #26336

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by delvh — APPROVED** [review]

File: templates/repo/settings/options.tmpl [hunk 1]
```diff
@@ -682,7 +682,7 @@
 				<div class="inline fields">
 					<label>{{.locale.Tr "repo.settings.admin_indexer_commit_sha"}}</label>
```
Comment (yardenshoham) [inline templates/repo/settings/options.tmpl]: Maybe we should include the label

Comment (CaiCandong) [reply templates/repo/settings/options.tmpl]: You mean this?
image

Comment (yardenshoham) [reply templates/repo/settings/options.tmpl]: Yes

**Review by yardenshoham — APPROVED** [review]

File: templates/repo/settings/options.tmpl (hunk 1 above)
Comment (CaiCandong) [reply templates/repo/settings/options.tmpl]: done cfd1e432517d5b5d6d91bad5f2c5530b9b135cdd

**Pull request merged**